etna/models/statsforecast.py

```diff
@@ -26,6 +26,7 @@
 from etna.models.mixins import PredictionIntervalContextIgnorantModelMixin
 
 StatsForecastModel = Union[AutoCES, AutoARIMA, AutoTheta, AutoETS, ARIMA]
+_DEFAULT_FREQ = object()
```

Comment:
Could you explain why we make it that way?

Reply:
Previous value `None` isn't correct because it is now a valid value for `freq`, so the default value should be different.

Here I wanted to create a sentinel object. I saw a [proposal in Python](https://peps.python.org/pep-0661/) about it, but it isn't going anywhere. So, I decided to do it using `object()`. The problem is that it doesn't work well with `typing` module. Do you have better alternatives in mind?

I used this solution in other places too. It we'll figure out a better way, I'll fix it everywhere.

Reply:
If the only reason of checking `_freq` is check for called `fit`. You can check other attributes like `_first_train_timestamp`

Reply:
It seems easier if all parameters are consistent and aren't set before `fit`.

In some places we check multiple parameters in `transform`/`forecast` because it helps mypy to deduct a correct type.